crates/uv-auth/src/credentials.rs

```diff
@@ -321,6 +321,29 @@ impl Credentials {
     }
 }
 
+/// Return a version of the URL with redacted credentials, allowing the generic `git` username (without a password)
+/// in SSH URLs, as in, `ssh://git@github.com/...`.
+pub fn redacted_url(url: &Url) -> Cow<'_, Url> {
+    if !tracing::enabled!(tracing::Level::DEBUG) {
+        return Cow::Borrowed(url);
+    }
```

Comment:
I'm actually a little more worried about this as a footgun (versus the need to repeatedly call `redacted` in the `tracing` macros). I think we should either remove this gating (always redact), or move the `redacted_url` calls into the `tracing` macros.

Reply:
Removed.

Reply:
It looks like this isn't removed, did you need to push still?

Reply:
Pushed